packages/kubernetes-1.24/kubelet-exec-start-conf

```diff
@@ -1,3 +1,7 @@
+[required-extensions]
+kubernetes = { version = "v1", helpers = ["any_enabled"] }
```

Comment:
Is it possible to make `any_enabled` a `std` helper or does it assume too much about the data shape to work for anything other than Kubernetes?